src/pages/tBTC/Bridge/Minting/KnowledgebaseLinks.tsx

```diff
@@ -0,0 +1,61 @@
+import { StackProps, VStack } from "@chakra-ui/react"
+import { FC } from "react"
+import { ExternalHref } from "../../../../enums"
+import { useFetchDepositDetails } from "../../../../hooks/tbtc"
+import {
+  BridgeProcessResources,
+  BridgeProcessResourcesItemProps,
+} from "../components/BridgeProcessResources"
+import {
+  TransactionHistory,
+  TransactionHistoryItemType,
+} from "../components/TransactionHistory"
+import bitcoinJuiceIllustration from "../../../../static/images/bitcoin-juice.png"
+import { useTbtcState } from "../../../../hooks/useTbtcState"
+
+export type KnowledgebaseLinksProps = { depositKey?: string } & StackProps
+
+const resources: BridgeProcessResourcesItemProps[] = [
+  {
+    title: "Token Contract",
+    link: ExternalHref.btcConfirmations,
+  },
+  {
+    title: "Bridge Contract",
+    link: ExternalHref.mintersAndGuardiansDocs,
+  },
```

Comment:
The links for those two contracts are wrong. Please see how we build those links based on the connected chain here:

https://github.com/threshold-network/bitcoin-on-base/blob/6dc91ef4fa6bd9794f26ef5dcecf06c39b6ebf66/src/components/tBTC/Links.tsx#L14-L43

Reply:
Ref commit: c599320c910f93b2a26f11341f79aa647a3d1f60